src/components/Admin/ManageQuestions.js

```diff
@@ -151,6 +151,39 @@ const ManageQuestions = ({ firebase }) => {
             setError(err);
           });
 
+        let levels = await firebase
+          .levelConfig()
```

Comment:
@vitorvicente Do you think it'd make sense for levelConfig to go inside a context that's always accessible? (like how we're planning to do for settings)

Reply:
Actually yeah it probably isn't a bad idea, I'm  gonna add it to that issue